src/actions/websocket.js

```diff
@@ -260,16 +260,12 @@ async function handleNewPostEvent(msg, dispatch, getState) {
     const currentUserId = users.currentUserId;
     const status = users.statuses[userId];
 
-    // Only fetch the profile and status if the post was made by someone else
-    // as well as making the DM channel visible
-    if (userId !== currentUserId) {
-        getProfilesAndStatusesForPosts([post], dispatch, getState);
-
-        // getProfilesAndStatusesForPosts only gets the status if it doesn't exist, but we also want it if
-        // the user does not appear to be online
-        if (status && status !== General.ONLINE) {
-            getStatusesByIds([userId])(dispatch, getState);
-        }
+    getProfilesAndStatusesForPosts([post], dispatch, getState);
```

Comment:
Really this should be renamed but I don't want to deal with that headache here, so I made a ticket https://mattermost.atlassian.net/browse/ABC-206